src/metabase/db/metadata_queries.clj

```diff
@@ -64,3 +63,30 @@
   [field]
   (-> (field-query field (ql/aggregation {} (ql/count (ql/field-id (u/get-id field)))))
       first first int))
+
+(defn db-id
```

Comment:
Something like `field->db-id` would be a more appropriate name for this, I think

Otherwise it's not clear it's only for Fields. Especially given the docstring